package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A utility for managing the `processinfo` folder that NYC uses.",
   "main": "index.js",
   "scripts": {
-    "test": "tap -j1 --cov test/*.js",
-    "snap": "cross-env TAP_SNAPSHOT=1 tap -j1 --cov test/*.js",
+    "test": "tap -j1 test/*.js",
+    "snap": "tap -j1 test/*.js",
```

Comment:
Could add `"jobs": 1` down in the `tap` section, and get rid of the glob.

Reply:
I can't get rid of the glob until the issue with Travis for Windows is figured out.  So I'll probably just leave it like this until tap@14.